### DISCUSSION TITLE
Conflict with AutofacBootstrapper and Fody Weaver

### DISCUSSION THREAD

**Discussion by drepamig** [canton7/Stylet#211] [Ideas]
Thanks for the great work!

I'm using your example AutofacBootstrapper with my project and was getting errors during load that stated:

```
Autofac.Core.Activators.Reflection.NoConstructorsFoundException: 'No accessible constructors were found for the type '[projectname]_ProcessedByFody'.'
```

After some head scratching, I realized I needed to filter that name out of the line in AutofacBootstrapper.cs:

```C#
// Original
builder.RegisterAssemblyTypes(GetType().Assembly).ExternallyOwned();

// Fixed
builder.RegisterAssemblyTypes(GetType().Assembly).Where(x=>!x.Name.Contains("ProcessedByFody")).ExternallyOwned();
```

I just wanted to share this in the case that someone else runs into this issue.

**Comment by canton7** [comment]
Thanks, nice detective work! I'll change that in the sample.
